include/deal.II/hp/refinement.h

```diff
@@ -382,7 +382,7 @@ namespace hp
 
     /**
      * Predict how the current @p error_indicators will adapt after refinement
-     * and coarsening has happened on the provided @p dof_handler, and write its
+     * and coarsening would have happened on the provided @p dof_handler, and write its
```

Comment:
maybe say "if refinement and coarsening were to happen". "would have happened" is past tense and considers something that might have happened in the past, but didn't. But you want to consider future tense: something that could happen in the future. "were to happen" is exactly the term you are looking for. (If you want to be extra explicit about the tense, you could say "were to happen in the future", though that means exactly the same.)